spec/acceptance/firewall_clusterip_spec.rb

```diff
@@ -7,32 +7,35 @@
     shell('ip6tables --flush; ip6tables -t nat --flush; ip6tables -t mangle --flush')
   end
 
-    describe 'clusterip' do
-      context 'cluster ipv4 test' do
-        it 'applies' do
-          pp = <<-EOS
-            class { '::firewall': }
-            firewall { 
-              '830 - clusterip test':
-                chain                 => 'FORWARD',
-                jump                  => 'CLUSTERIP',
-                destination           => '1.1.1.1',
-                iniface               => 'eth0',
-                clusterip_new         => true,
-                clusterip_hashmode    => "sourceip",
-                clusterip_clustermac  => "01:00:5E:00:00:00",
-                clusterip_total_nodes => "2",
-                clusterip_local_node  => "1",
-                clusterip_hash_init   => "1337",
-            }
-          EOS
+    # Have to comment out el-7 as the interface name is unknown
+    if default['platform'] !~ /el-7/
```

Comment:
please add a `else pending "EL7 interface names are not stable. see MODULES-2124"` here